packages/nextjs/app/layout.tsx

```diff
@@ -1,12 +1,13 @@
 import "@rainbow-me/rainbowkit/styles.css";
 import { ScaffoldEthAppWithProviders } from "~~/components/ScaffoldEthAppWithProviders";
 import { ThemeProvider } from "~~/components/ThemeProvider";
+import projectConfig from "~~/project.config";
 import "~~/styles/globals.css";
 import { getMetadata } from "~~/utils/scaffold-eth/getMetadata";
 
 export const metadata = getMetadata({
-  title: "Scaffold-ETH 2 App",
-  description: "Built with 🏗 Scaffold-ETH 2",
+  title: `${projectConfig.name} App`,
+  description: projectConfig.description,
```

Comment:
We have `getMetadata` also on `/blockexplorer` and `/debug` pages, and I don't think we need to change that. For me, the default description here is in line with other metadata descriptions, so I think we don't need to move description to variable